catalogue/webapp/components/VisuallySimilarImagesFromApi/VisuallySimilarImagesFromApi.tsx

```diff
@@ -1,41 +1,66 @@
-import { font } from '@weco/common/utils/classnames';
-import { Image as ImageType } from '@weco/common/model/catalogue';
 import { FunctionComponent, useEffect, useState } from 'react';
 import styled from 'styled-components';
-import { getImage } from '../../services/catalogue/images';
+
+import { font } from '@weco/common/utils/classnames';
+import { Image as ImageType } from '@weco/common/model/catalogue';
+import { getImage } from '@weco/catalogue/services/catalogue/images';
 import Space from '@weco/common/views/components/styled/Space';
 import { useToggles } from '@weco/common/server-data/Context';
-import IIIFImage from '../IIIFImage/IIIFImage';
+import IIIFImage from '@weco/catalogue/components/IIIFImage/IIIFImage';
+import LL from '@weco/common/views/components/styled/LL';
 
 type Props = {
   originalId: string;
   onClickImage: (image: ImageType) => void;
 };
 
-const Wrapper = styled.div`
+const Wrapper = styled(Space).attrs({
+  v: { size: 's', properties: ['margin-bottom', 'margin-top'] },
+})`
   display: flex;
+  align-items: center;
   flex-wrap: wrap;
-  align-items: flex-end;
+  min-height: 120px;
+
+  ${props => props.theme.media('large')`
+    flex-wrap: nowrap;
+  `}
+
+  a {
+    flex: 1 0 auto;
+
+    ${props => props.theme.media('medium')`
+      flex: 0 1 auto;
+    `}
+  }
 
   img {
     margin-right: 10px;
-    margin-bottom: 10px;
-    height: auto;
-    max-height: 120px;
-    max-width: 190px;
     width: auto;
+    max-height: 120px;
+
+    ${props => props.theme.media('large')`
+      max-width: 150px;
+      width: calc(100% - 10px);
+    `};
   }
 `;
 
+const LoaderWrapper = styled.div`
+  height: 120px;
+`;
+
 const VisuallySimilarImagesFromApi: FunctionComponent<Props> = ({
   originalId,
   onClickImage,
 }: Props) => {
   const [similarImages, setSimilarImages] = useState<ImageType[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
Not necessarily for now / this PR, but we have found these kind of async state booleans to become a real nightmare and now much prefer something like
```typescript
type State = 'loading' | 'error' | 'success' | 'initial';
const [requestState, setRequestState] = useState<State>('initial');
```
because it prevents 'impossible' states from occurring - and feels much more natural in Typescript

Reply:
That does look a lot cleaner! We do use it across, maybe this could be done at a higher level in a different ticket - I'm looking into using load states a bit more so happy to create one for our backlog.

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/issues/8832

Reply:
Nice! Apologies if this is a concept you're already familiar with but this comes from the idea that we should aim to "make invalid states impossible": there are a whole load of bugs in all kinds of software that stem from them getting into a state that doesn't make sense, so by making that state impossible we eliminate the bugs. https://kevinmahoney.co.uk/articles/applying-misu/ has some nice examples, and https://kentcdodds.com/blog/make-impossible-states-impossible is this specific case.

Reply:
Don't apologise, thanks for sharing these!

Reply:
@jamieparkinson I've made the changes you suggested for this bit, if you wouldn't mind having a quick look and confirming it's what you'd meant?

Reply:
Fab!